server/test/integration/routes/projects.get.spec.js

```diff
@@ -205,7 +206,7 @@ describe('integration: projects with open event by default', () => {
         '"Participant 1 Gender"',
         '"Participant 1 Special requirements"',
       ]);
-      const userCSVColumns = ['"First name"', '"Last name"', '"Dob"', '"Gender"', '"Special requirements"'].concat(projectCSVColumns);
+      const userCSVColumns = ['"Project ID"', '"First name"', '"Last name"', '"Dob"', '"Gender"', '"Special requirements"'].concat(projectCSVColumns);
```

Comment:
`projectCSVColumns.slice(1)` will exclude the `ProjectId` from the array : it's duplicated, hence the failing test